website/api/views.py

```
@@ -1,7 +1,7 @@
 import json
 import smtplib
 import uuid
-from datetime import datetime
+from datetime import datetime, timedelta
```

Ruff (0.8.2)

`datetime.timedelta` imported but unused

Remove unused import: `datetime.timedelta`

(F401)